templates/user-profile.tpl.php

```diff
@@ -98,8 +98,8 @@
     <h2>Submit Your Work</h2>
       <div style="line-height: 2.0; font-size: larger;">
           <ul style="list-style: none;">
-              <li style="padding-bottom:10px;"><a href='http://trace.tennessee.edu/submit_research.html'>Submit Faculty and Graduate Student Research and Creative Work</a></li>
-              <li style="padding-bottom:10px;"><a href='/islandora/object/utk.ir%3Atd/manage/overview/ingest'>Submit Graduate Theses and Dissertations</a></li>
+              <li style="padding-bottom:10px;"><a href='https://trace.tennessee.edu/submit_research.html'>Submit Faculty and Graduate Student Research and Creative Work</a></li>
+              <li style="padding-bottom:10px;"><a href='https://trace.tennessee.edu/utk_grad/'>Submit Graduate Theses and Dissertations</a></li>
```

Comment:
```suggestion
              <li style="padding-bottom:10px;"><a href='https://trace.tennessee.edu/utk-grad/'>Submit Graduate Theses and Dissertations</a></li>
```

Reply:
:facepalm: thank you

Reply:
Just FYI:  you don't have to make these changes locally; you can use the commit suggestion button.